docs/zh/architecture.md

```diff
@@ -20,8 +20,7 @@ AWS 云帐户并提供三个 URL。一个用于**可视化**。
 02. 笔记本附带针对不同药物研发问题的示例代码，如分子展开、分子模拟等。
 用户可以学习如何利用经典计算或通过访问
 [Amazon Braket](https://aws.amazon.com/braket/)
-利用量子计算研究这些问题。请参考[动手实验][workshop-background]
-中的指南。
+利用量子计算研究这些问题。请参考[动手实验](workshop/background.md)中的指南。
```

Comment:
if using direct link, removing the link anchors at end of file

Reply:
moved this problem to the architecture issue (#65) by liu yong